### PR TITLE
Bugfix/2722 background colors my habits

### PR DESCRIPTION
**Pull request by idzhersh**
Changed background colors according to requirements
###Before

###After
Screenshot (580)

### CODE REVIEW

File: src/app/component/layout/components/footer/footer.component.scss [hunk 1]
```diff
@@ -79,6 +78,10 @@ li {
     padding: 0;
   }
 
+  .footer_bottom-part {
+    float: left;
```
Comment (paslavskyi9roman) [inline src/app/component/layout/components/footer/footer.component.scss:82]: I would suggest not using "float" property since it's kinda depricated

Comment (idzhersh) [reply src/app/component/layout/components/footer/footer.component.scss:86]: It works fine...I think It does not matter

Comment (paslavskyi9roman) [reply src/app/component/layout/components/footer/footer.component.scss:86]: Floats should not be used for layout anymore (though you can still use them for the original purpose - floating text around images).

Flexbox is now widely supported and is better for layout.

**Review by yuriiLisunov — APPROVED** [review]

**Review by SofiiaBaziv — APPROVED** [review]

**Comment by JimHolman1199** [comment]
Fix conflicts

**Comment by idzhersh** [comment]
> Fix conflicts

fixed

**Review by AlekOK — APPROVED** [review]

**Pull request merged**